### ISSUE TITLE
Add template html tag

### ISSUE THREAD

**Issue by wout** [luckyframework/lucky#1154]
I'm using alpinejs and find myself using quite a few `<template>` tags:

```crystal
tag "template", x_if: "ready" do
  para "I'm ready"
end
```

It would be convenient to be able to write:

```crystal
template, x_if: "ready" do
  para "I'm ready"
end
```

**Comment by paulcsmith** [comment]
100% 👍 

Also, how are you liking Alpine!? I'm planning to use that on Fusion :)

**Comment by jwoertink** [comment]
closed by #1164
